website/docs/tutorials/basic/running_your_app/2_multirun.md

```diff
@@ -21,6 +25,27 @@ The following sweeps over all combinations of the dbs and schemas.
 ```
 The printed configurations have been omitted for brevity.
 
+You can also define sweeping in the input configs by overriding
+`hydra.sweeper.params`. Using the above example, the same multirun could be achieved via the following config.
+
+```yaml
+hydra:
+  sweeper:
+    params:
+      db: mysql,postgresql
+      schema: warehouse,support,school
+```
+
+The syntax are consistent for both input configs and commandline overrides. The commandline will override the definition 
+in input configs. If we run the same application with the above input config and a new commandline override:
```

Comment:
I think that the word "override" is overloaded (as here it refers both to the sweeper overriding the config settings in different jobs, and to the command line sweep taking precedence over the the sweep from the input config).
```suggestion
The syntax are consistent for both input configs and commandline overrides.
If a sweep is specified in both an input config and at the command line,
then the commandline sweep will take precedence over the sweep defined 
in the input config. If we run the same application with the above input config and a new commandline override:
```

Reply:
sg! thanks for the edits